### ISSUE TITLE
Symbols

### ISSUE THREAD

**Issue by lilactown** [clavascript/clavascript#5]
Should Clojure symbols map to [Symbols](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol), some custom type, or strings like keywords?

**Comment by borkdude** [comment]
Not sure yet. Let's keep this open until we can make an informed decision.

**Comment by borkdude** [comment]
In contrast to Clojure, Symbols in JS are identical (if created with /for and the same string). In Clojure, symbols are not interned and can care metadata. I think the are too different to map onto each other.

**Comment by lilactown** [comment]
@borkdude that's only if you use `Symbol.for` - if you use `Symbol(string)` it will return a new symbol value each time.

Perhaps we should wait until we figure out how we're going to handle metadata for objects and other types

**Comment by borkdude** [comment]
Indeed! Perhaps we can implement metadata using the protocol which uses the Symbol like you suggested.
